tools/build/workspace.bzl

```diff
@@ -58,7 +58,7 @@ def gapid_dependencies(android = True, mingw = True, locals = {}):
         locals = locals,
         organization = "google",
         project = "protobuf",
-        commit = "815ff7e1fb2d417d5aebcbf5fc46e626b18dc834", # Head of 3.8.x branch
+        commit = "815ff7e1fb2d417d5aebcbf5fc46e626b18dc834",  # Head of 3.8.x branch
```

Comment:
Extra space?

(this may be due to some auto-formatter of .bzl files)

Reply:
yes, they are done automatically by the buildifier

Reply:
Is it possible to isolated them in a separate PR?

I'm pushy on this to avoid later `git log` / `git blame` to have these formatting changes linked to a commit that is about dumping an image in the virtual swapchain.